src/main/java/com/incognia/onboarding/SignupAssessment.java

```diff
@@ -10,6 +12,7 @@ public class SignupAssessment {
   UUID id;
   UUID requestId;
   Assessment riskAssessment;
+  Set<Reason> reasons;
```

Comment:
Does the library map the JSON response into this object?

Reply:
If that is the idea, we shouldn't do that or we would break the clients when adding a new reason.

Reply:
The reason object contains only two string fields, not ReasonCode and ResonSource

Reply:
![image](https://user-images.githubusercontent.com/9062839/168837425-dc682320-5bab-4aa1-a20d-745e1273f522.png)

Reply:
Oh, my bad!